app/models/auction_status.rb

```diff
@@ -19,6 +19,10 @@ def expiring?
     available? && auction.ended_at < (Time.current + 12.hours)
   end
 
+  def work_in_progress?
```

Comment:
I was going to suggest that potentially, but we are kinda getting to a point where we are recapitulating a state machine too ;)

One thing that is annoying is that we have currently 3 different places where `status` is described for an auction:

1. the `status` enum column
2. the AuctionStatus class (normally just derived from other fields)
3. The `c2_status` field

I wasn't quite sure where to put this at first and thought for a while before I felt like it should go with the `future?` and other methods that are derived from the values of other fields in the auction. But we could consider some renaming (like AuctionStatus -> AuctionAvailability) that might mitigate some confusion